samples/app/src/main/java/com/example/app/userkeys/UserKeysRoute.kt

```diff
@@ -66,9 +66,14 @@ fun UserKeysRoute(viewModel: UserKeysViewModel, openDrawer: () -> Unit) {
         )
     }
 
+    userKeysState.exception?.apply {
+        Alert(exception = this)
+        userKeysState.exception = null
+    }
+
     Column(modifier = Modifier.padding(16.dp)) {
         Topbar(heading = "User Keys", openDrawer)
```

Comment:
Heading can be like this 

User Keys - DeviceBind 

or like

DeviceBind - User Keys

Reply:
Updated